### PR TITLE
Correction Factor Adjustment - Fixes #40

### LINKED ISSUE #40: Apply correction factor for small particle data measurements
Side-by-side deployments of other particle measurement instruments next to those in the network and determined there is a correction factor for PM1, P2.5, and PM10 data, and its **17x**. There’s a consistent and systematic undercounting issue with the optical measurement (as opposed to a mass-based measurement). We need to increase these values by that factor in the backend.

### PR DESCRIPTION
**Pull request by Inkering**
Working on fixing the correction factor. The bar component could use a little refactoring as it currently isn't as simple as the graph to implement modular corrections.

still adding commits!

### CODE REVIEW

File: src/PollutantsComponent.js [hunk 1]
```diff
@@ -50,8 +50,13 @@ function PollutantsComponent(props) {
             return data;
 
         for (let dataPoint of device.graph) {
+            let currPollutant = PollutantByID(pollutantId);
             if (typeof dataPoint[pollutantId] != 'undefined')
```
Comment (hsharriman) [inline src/PollutantsComponent.js:54]: I might be blind but is this if statement doing anything here? I don't see any enclosing `{}`

**Review by davidt315 — DISMISSED** [review]

File: src/PollutantsComponent.js (hunk 1 above)
Comment (Inkering) [reply src/PollutantsComponent.js:54]: Good catch, remnant of old code I think. Will adjust.

**Review by wFairmanOlin — APPROVED** [review]

**Review by wFairmanOlin — APPROVED** [review]

File: src/PollutantBarComponent.js [hunk 2]
```diff
@@ -112,10 +112,16 @@ function PollutantBarComponent(props) {
     };
 
     const getNumericalVal = () => {
+        if (pollutant === 'PM25' || pollutant === 'PM10') {
+            // TODO: modularize with correctionFactor prop in Utils.js
```
Comment (anushadatar) [inline src/PollutantBarComponent.js:119]: I'm cool with not doing this change right now, but here's an [Issue](https://github.com/airpartners/aq-web-client/issues/50) about this TODO.

**Review by anushadatar — APPROVED** [review]
LGTM - thanks for doing this Dieter!

**Pull request merged**